test/paper-dialog-behavior.html

```diff
@@ -178,6 +197,42 @@ <h2 id="header">Dialog</h2>
           });
         });
 
+        test('multiple modal dialogs opened, handle focus change', function(done) {
+          var dialogs = fixture('multiple');
+          var spy = sinon.spy(dialogs[0], '_boundOnFocus');
+          runAfterOpen(dialogs[0], function() {
+            runAfterOpen(dialogs[1], function() {
+              var button = Polymer.dom(dialogs[1]).querySelector('[extra]');
+              button.click();
+              setTimeout(function() {
+                // Should not enter in an infinite loop.
+                assert.notEqual(spy.callCount, 0, 'focus changed at least once.');
+                // On IE focus changes twice.
```

Comment:
This is caused by the fact that we set [`useCapture`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) to `true` in here (the third param) https://github.com/PolymerElements/paper-dialog-behavior/blob/master/paper-dialog-behavior.html#L197

Here a quick example with just two buttons, also in chrome we see that focus callback is called twice http://jsbin.com/xobeqe/edit?html,output

Do you think it's safe to set `useCapture` to `false`?